### PR TITLE
docs: add MLFlow in ai sample time series forecasting

### PR DESCRIPTION
**Pull request by lhrotk**
## Related Issues/PRs

None

## What changes are proposed in this pull request?

Add MLFlow logging and loading to Time Series Forecasting sample notebook

## How is this patch tested?

- [x] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [x] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [x] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

AB#1966880

### CODE REVIEW

File: notebooks/community/aisamples/AIsample - Time Series Forecasting.ipynb [hunk 1]
```diff
@@ -398,6 +398,59 @@
...
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "# log the model and parameters\n",
+                "model_name = f\"{EXPERIMENT_NAME}-{uuid.uuid4()}\"\n",
```
Comment (serena-ruan) [inline notebooks/community/aisamples/AIsample - Time Series Forecasting.ipynb:436]: Can we remove this uuid as run_id is already random?

Comment (lhrotk) [reply notebooks/community/aisamples/AIsample - Time Series Forecasting.ipynb]: You are correct, random name is useless. Model name changed

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**